apps/docs/components/preview/TypographyPreview.tsx

```diff
@@ -0,0 +1,35 @@
+import "@hopper-ui/tokens/fonts.css";
+import "./preview.css";
+
+interface TypographyPreviewProps {
+    values?: TypographyValues;
+}
+
+interface TypographyValues {
+    lineHeight?: string;
+    fontWeight?: string;
+    fontSize?: string;
+    fontFamily?: string;
+}
+
+interface TypographyPreviewElemProps {
+    className?: string;
+    content?: React.ReactNode;
+    style?: React.CSSProperties;
+}
+
+const TypographyPreview = ({ values }: TypographyPreviewProps) => {
+    let TypographyPreviewElem: TypographyPreviewElemProps = {};
+
+    const sampleText = "Aa";
```

Comment:
you should inline this, not put in a variable it's useless

Reply:
Made sense in the Table as there was many switch cases.